entity-management-web/src/main/resources/entitymanagement.properties

```diff
@@ -12,13 +12,20 @@ entitymanagement.api.version=0.0.1-SNAPSHOT
 
 entity.environment=development
 
-europeana.thumbnail.urlPrefix=https://api.europeana.eu/api/v2/thumbnail-by-url.json?type=IMAGE&uri=
-
 # Resource name embedded in JWT token
 authorization.api.name=entities
 
-#solr query for computing enrichments count
-entitymanagement.solr.searchapi.enrichments.query="%s" AND contentTier:(2 OR 3 OR 4)
+europeana.thumbnail.urlPrefix=https://api.europeana.eu/api/v2/thumbnail-by-url.json?type=IMAGE&uri=
+
+
+# URL prefix for querying enrichments count
+europeana.searchapi.urlPrefix=
+
+# query for computing enrichments count
+europeana.searchapi.enrichments.query="%s" AND contentTier:(2 OR 3 OR 4)
```

Comment:
I would recommend to have the contentTier filter as separate property as this is not used for organizations